.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -44,4 +44,5 @@ Closes #ISSUE_NUMBER.
 ## Checklist
 <!--- Go over all the following points, and make sure you've done anything necessary -->
 - [ ] I have added tests to cover my changes, if necessary.
+- [ ] I have added translations for new strings, if necessary.
 * I have updated the documentation accordingly, if necessary.
```

Comment:
```suggestion
- [ ] I have updated the documentation accordingly, if necessary.
```

Reply:
Going the opposite direction, I think?

Reply:
I would advocate to use checkboxes for all items under that category. It would require action to take to ensure items are performed.

Reply:
@anthoula the team intentionally changed to use bullets in the past, but then swapped back only the test line checkbox as a way to ensure folks paid attention to it. That said, you'll find that most people ignore it, which means that we end up with github telling us that folks have `0 out of 1` checkboxes checked, as in the image I linked above.

My preference would be to not use checkboxes unless we're going to start holding folks to actually checking them instead of ignoring them. I will leave it to the team.

Reply:
I personally want them to address the checkboxes because they seem to be important. But how to handle cases where a PR has nothing to do with Docs for instance? Honestly, I don't remember if we had this conversation before about checkboxes vs bullet points.